deploy/src/main/deploy/helm/templates/prometheus/prometheus-config.yaml

```diff
@@ -24,6 +24,9 @@ data:
     {{- end }}
     {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-amqp-vertx") | nindent 4 }}
     {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-http-vertx") | nindent 4 }}
+    {{- if .Values.adapters.lora.enabled }}
+      {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-lora") | nindent 4 }}
```

Comment:
the service names seems to be `adapter-lora-vertx`, doesn't it?

Reply:
yes, you're right. Changed it